src/libs/Navigation/AppNavigator/Navigators/Overlay.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import {Animated, View} from 'react-native';
+// TODO: migrate
```

Comment:
Lets not leave todos in code but instead create an issue for it and link it in a comment on a PR

Reply:
I just migrated the overlay as well. We will never really see the overlay on mobile platforms, as the screens are opening above the current screen, so I made the overlay none mobile/native only